app/src/App.vue

```diff
@@ -1,27 +1,31 @@
 <template>
   <div id="app">
-    <HelloWorld msg="Initial Commit"/>
+    <MainPage/>
```

Comment:
add here your Header component as we need to render it on each page and after it add router-vue. It will render needed page for each route